server/remote_cache/cachetools/cachetools.go

```diff
@@ -101,6 +106,10 @@ func GetBlobChunk(ctx context.Context, bsClient bspb.ByteStreamClient, r *digest
 		}
 		written += int64(n)
 	}
+	computedDigest := fmt.Sprintf("%x", checksum.Sum(nil))
```

Comment:
should the actual checksum go in `GetBlob` since the chunk digest won't necessarily match the full blob digest? (or only do the checksum if Offset=0 and Limit=0)

Reply:
Ah yeah that's a good point.

Reply:
Done.